dataminer-overview/Partner_Program/About_Partner_Program.md

```diff
@@ -10,10 +10,14 @@ The moment you become our partner, you start receiving benefits.
 
 - Buy DataMiner Licenses, Usage-based Services, and Support Services at reduced rates
 
-- Access lower certification and training fees
+- Early access to roadmap discussions
 
 - Referrals
 
+- Updated Go-To-Market Strategies
```

Comment:
OK to change this to sentence case? There doesn't seem to be a reason why title case is needed here.
```suggestion
- Updated go-to-market strategies
```